CHANGELOG.asciidoc

```diff
@@ -18,7 +19,7 @@
 // end::list[]
 
 // tag::includes[]
-include::./changelogs/head.asciidoc[]
+include::./changelogs/8.16.asciidoc[]
```

Comment:
head needs to stay, it should look like this: https://github.com/elastic/apm-server/pull/14381/files#diff-cb0601935fcaac3fea29b215282428b964b8b6e1bd5a6b8c68e84d651633c9e3R22-R23

Reply:
Fixed in 65f8ee954fc90ef5795166223ebf7ca943ae399e